misc/config/config.nydus.ref.yaml

```diff
@@ -34,6 +34,8 @@ provider:
       threshold: 1000MB
   # remote cache record capacity of converted layers, default is 200.
   cache_size: 200
+  # remote cache version
```

Comment:
remote cache version, the remote cache will be discarded if uses a version not matched with this.

Reply:
`remote cache version, cache in remote must match the specified version, or discard cache.`